images/win/toolsets/toolset-2019.json

```diff
@@ -374,5 +374,31 @@
             "3.1",
             "5.0"
         ]
+    },
+    "choco": {
+        "common_packages": [
+            { "name": "7zip.install"},
+            { "name": "azcopy10"},
+            { "name": "Bicep"},
+            { "name": "gitversion.portable"},
+            { "name": "innosetup"},
+            { "name": "jq"},
+            { "name": "NuGet.CommandLine"},
+            { "name": "openssl.light"},
+            { "name": "packer"},
+            { "name": "strawberryperl"},
```

Comment:
There was an issue with strawberry perl, openssl, and git in the PATH. Need to make sure that nothing is broken after these changes
https://github.com/actions/virtual-environments/pull/548

Reply:
Compared the PATH in current image and after my changes, the order of directories for strawberry perl, openssl, and git is the same

Reply:
Thanks for the update!

Reply:
The PATH has actually changed a bit, but it looks like nothing critical:
![image](https://user-images.githubusercontent.com/48208649/124755795-76d69580-df34-11eb-842e-bddb603859f5.png)
All the applications are in the same places judging by `Get-Command -CommandType Application`